### DISCUSSION TITLE
我的世界服务器

### DISCUSSION THREAD

**Discussion by HYhaoil** [share121/inter-knot#2100] [其他]
还在为网易我的世界服务器烂不能愉快和好友开黑吗，还在因为租赁服贵只能一个人开存档联机导致不能随时随地愉快玩耍而发愁吗，我的世界粘液科技生存公益服务器正在开荒，java版本1.20.1，欢迎大家加入交流游玩以及分享资源，qq群聊497601815
![IMG_20240814_185407](https://github.com/user-attachments/assets/b0a26a61-4a6f-4d68-b687-4e97a5f054a1)

**Comment by Enigma-Soul** [comment]
不是正版我不玩

**Comment by HYhaoil** [comment]
正版验证ON
________________________________
发件人: Enigma_Soul ***@***.***>
发送时间: 2024年8月14日 18:56
收件人: share121/inter-knot ***@***.***>
抄送: 椰子乱飞 ***@***.***>; Author ***@***.***>
主题: Re: [share121/inter-knot] 我的世界服务器 (Discussion #2100)


不是正版我不玩

―

**Reply by Enigma-Soul** [reply]
没有OP我不玩

**Comment by HYhaoil** [comment]
我不玩o神

________________________________
发件人: Enigma_Soul ***@***.***>
发送时间: 2024年8月14日 19:00
收件人: share121/inter-knot ***@***.***>
抄送: 椰子乱飞 ***@***.***>; Author ***@***.***>
主题: Re: [share121/inter-knot] 我的世界服务器 (Discussion #2100)


没有OP我不玩

―

**Reply by Enigma-Soul** [reply]
OP权限

**Comment by FNN66** [comment]
宁愿java30帧，也不想玩冈易的T_T

**Reply by Enigma-Soul** [reply]
不是，30？，你加没加优化模组啊，

**Reply by FNN66** [reply]
优化模组肯定是要加的，你就当我是在讽刺网易的吧，网易做的怎么样咱也清楚

**Comment by SuiHanjun** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
这还能有我的世界玩家

**Comment by XokoukiguraX** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

好
